### PR TITLE
Use the router to get the correct path from the connect facebook check route

### PR DESCRIPTION
**Pull request by alexander-schranz**
Prevent problems like: https://github.com/knpuniversity/oauth2-client-bundle/issues/39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ class MyFacebookAuthenticator extends SocialAuthenticator
     public function supports(Request $request)
     {
         // continue ONLY if the URL matches the check URL
-        return $request->getPathInfo() == '/connect/facebook/check';
+        return $request->getPathInfo() == $this->router->getRouteCollection()->get('connect_facebook_check')->getPath();
```
Comment (bocharsky-bw) [inline README.md]: Why don't just `$this->router->generate('connect_facebook_check')`?

Comment (alexander-schranz) [reply README.md]: would add a host when the route has a host defined or?

Comment (weaverryan) [reply README.md]: It shouldn't add a host, but I'm actually not sure if you have the host defined - good point :).

But, there another option anyways, where we look specifically at the *route* name:

```php
return $request->attributes->get('_route') == 'connect_facebook_check';
```

Do you guys like that better?

Comment (alexander-schranz) [reply README.md]: Looks good for me did update the PR

**Pull request merged**